### ISSUE TITLE
Always show "Error: The start row index must be lesser than the end row index"

### ISSUE THREAD

**Issue by IvyGongoogle** [PaddlePaddle/PaddleDetection#1168]
Hello, when I use `  cascade_rcnn_mobilenetv3_fpn_640.yml` to finetune my own dataset like:
```
python -u tools/train.py -c configs/mobile/cascade_rcnn_mobilenetv3_fpn_640.yml \
              -o pretrain_weights=https://paddlemodels.bj.bcebos.com/object_detection/cascade_rcnn_mobilenetv3_fpn_640.tar \
              finetune_exclude_pretrained_params=['cls_score','bbox_pred'] \
              --eval 
```
but after running about 2000 iters, it always show error:

```
Error Message Summary:
----------------------
Error: The start row index must be lesser than the end row index.
  [Hint: Expected begin_idx < end_idx, but received begin_idx:0 >= end_idx:0.] at (/paddle/paddle/fluid/framework/tensor.cc:83)
  [operator < multiclass_nms > error]
```

envs: python3.7, paddlepaddle 1.8.2.post97, paddleDetection-0.3

How to fix this? Can you give some advises? 
Thanks.

**Comment by littletomatodonkey** [comment]
Can you show the complete log file during the training process? Maybe the loss is abnormal, If you finetune it on your own dataset, smaller learning rate is recommended(1/10 of the base lr in the config file)

**Comment by IvyGongoogle** [comment]
@littletomatodonkey  Thanks for your prompt reply.  
Take `cascade_rcnn_mobilenetv3_fpn_640.yml` for example, the default `base_lr` is 0.01, I read the docs and know it is for 8 gpus. So in single gpu, I first divide the`base_lr` by 8 with the default `batch_size: 2` [here](https://github.com/PaddlePaddle/PaddleDetection/blob/release%2F0.4/configs/mobile/cascade_rcnn_mobilenetv3_fpn_640.yml#L146), now it becomes 0.00125. Then because it is fine-tuning, I should divide 0.00125 by 10 again, now the `base_lr` becomes 0.000125. Am I correct?

**Comment by littletomatodonkey** [comment]
Yes, you are right, actually the base lr in `cascade_rcnn_mobilenetv3_fpn_640 .yml` should be 0.02(we will fix it soon), so you `base lr` should be 0.00025.
